com.civicrm.hrjobroles/views/include/job_role_panel.html

```diff
@@ -64,7 +64,7 @@
                 </div>
                 <div class="form-group" ng-init="jobroles.initData(job_roles_data.id, 'job_contract_id', job_roles_data.job_contract_id)">
                   <label for="contract" class="col-sm-4 control-label">Contract:</label>
-                  <div class="col-sm-8">
+                  <div class="col-sm-6">
```

Comment:
The columns width was kept from the original design. It might need some rework but for now I've decided to keep it as is (the scope of the ticket was switching to a different them, not to fix/amend the layout of job roles)